website/docs/r/kubernetes_cluster.html.markdown

```diff
@@ -223,6 +223,8 @@ A `default_node_pool` block supports the following:
 
 -> **NOTE:** This requires that the `type` is set to `VirtualMachineScaleSets` and that `load_balancer_sku` is set to `Standard`.
 
+* `only_critical_addons_enabled` - (Optional) Enabling this option will taint default node pool with `CriticalAddonsOnly=true:PreferNoSchedule` taint. Changing this forces a new resource to be created.
```

Comment:
_minor_ but could we sort this alphabetically to match the rest of this file (move this a few lines down)

Reply:
Sigh, I actually did sort it alphabetically to only then correct the fact that it wasn't `critical_addons_enabled`, but `only_critcal_addons_enabled` :D